assets/roks-metrics/roks-metrics-deployment.yaml

```diff
@@ -0,0 +1,52 @@
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: metrics
+  namespace: openshift-roks-metrics
+spec:
+  replicas: 1
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 2
+      maxUnavailable: 1
+  selector:
+    matchLabels:
+      app: metrics
+  template:
+    metadata:
+      labels:
+        app: metrics
+{{ if .RestartDate }}
+      annotations:
+        openshift.io/restartedAt: "{{ .RestartDate }}"
+{{ end }}
+    spec:
+      tolerations:
+        - key: "multi-az-worker"
+          operator: "Equal"
+          value: "true"
+          effect: NoSchedule
+      affinity:
+        podAntiAffinity:
```

Comment:
This is intentionally left blank?

Reply:
we don't need the affinity part, will remove